nodes/modifier_change/opensubdivide.py

```diff
@@ -0,0 +1,111 @@
+# ##### BEGIN GPL LICENSE BLOCK #####
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software Foundation,
+#  Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# ##### END GPL LICENSE BLOCK #####
+
+import bpy
+from bpy.props import IntProperty
+from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.data_structure import updateNode, match_long_repeat
+
+enable_module = False
+try:
+    import pyOpenSubdiv
+    from pyOpenSubdiv.pysubdivision import pysubdivide    
+    enable_module = True
+except ModuleNotFoundError:
+    enable_module = False
+
+from itertools import chain 
+import traceback 
+class SvOpenSubdivideNode(bpy.types.Node,SverchCustomTreeNode):
+    bl_idname = "SvOpenSubdivideNode"
+    bl_label = "OpenSubdiv"
+    bl_icon = 'OUTLINER_OB_EMPTY'
+    sv_icon = None 
+    
+    maxSubdivision = 5 # Creates a self.maxSubdivision attribute
+    maxlevel : IntProperty(name='level',default=0,min=0,max=maxSubdivision,update=updateNode)
```

Comment:
```suggestion
```

When you use `socket.use_prop` it means that socket uses its own property and there is no need to create them inside node. The problem is that it's impossible to set maximum value for socket property. Blender has API for that but with nodes it does not work. Probably it can goes without maximum limit because the value can go via socket any way where there is no any limits. Or you can return everything back as it was (using `prop_name`) =)